cli/cmd/generate.go

```diff
@@ -66,6 +66,7 @@ subcommands.`,
 
 	cmd.Flags().StringP("policy", "p", rulesFilename, "path to policy (.rego) file")
 	cmd.Flags().StringP("settings", "s", settingsFilename, "path to settings (.json) file")
+	cmd.Flags().StringP("cache-path", "c", cachePath, "path to cache for the cache (.json) file containing the image layers")
```

Comment:
I think we should give the flag a prefix like "layers-" or "genpolicy-". Contrast has itself a cache directory (currently controlled via env vars) which might be confusing.

Reply:
Yeah, I also stumbled over the "other" cache dir.  Since it may be that we need to pass more flags to genpolicy eventually, I'll go with `genpolicy-`.